mne/images/README.rst

```diff
@@ -0,0 +1,40 @@
+.. -*- mode: rst -*-
+
+
+Documentation
+=============
+
+The images are used in ``mne/viz/_brain`` as icons in the toolbar of ``_TimeViewer``.
+It is necessary to compile those images into a resource file for proper use by
+the application.
+
+The resource configuration file ``mne.qrc`` describes the location of the resources
+in the filesystem and also defines aliases for their use in the code.
+
+To automatically generate the resource file in ``mne/viz/_brain``:
+
+.. code-block:: bash
+
+    pyrcc5 -o mne/viz/_brain/resources.py mne.qrc
+
+
+Patching
+========
+
+The output file imports ``PyQt5`` globally, which is not consistent with MNE core
```

Comment:
Rather than all of this, can you nest the import of `resources.py` wherever it's actually used in the MNE code?

Reply:
I will try that right now